core/src/components/select/select.ionic.scss

```diff
@@ -9,14 +9,18 @@
   --background: #{globals.$ion-primitives-base-white};
   // TODO(ROU-10778, ROU-10875): Sync the color names to the design system of
   // ios and md. This will allow us to have a single color map.
-  --border-color: #{globals.current-color(neutral)};
+  --border-color: #{globals.$ion-primitives-neutral-500};
   --border-width: #{globals.$ion-border-size-025};
+  --focus-ring-color: #{globals.$ion-border-focus-default};
   --padding-start: #{globals.$ion-space-400};
   --padding-end: #{globals.$ion-space-400};
   --padding-top: #{globals.$ion-space-300};
   --padding-bottom: #{globals.$ion-space-300};
   --placeholder-color: #{globals.$ion-primitives-neutral-800};
   --placeholder-opacity: 1;
+  --highlight-color-focused: #{globals.$ion-border-focus-default};
+  --highlight-color-valid: #{globals.$ion-semantics-success-900};
+  --highlight-color-invalid: #{globals.$ion-semantics-danger-800};
```

Comment:
Why was this value changed? The ticket is supposed to be for focus only. I would recommend making a separate PR.

Reply:
But it is related to focus as well. When the user focus on a valid or invalid select, the focus color should have the expected semantic color.